docs/content/users/topics/sharing.md

```diff
@@ -46,6 +46,14 @@ This set of steps assumes an ngrok subdomain `mg2`:
 * Run `bin/magento setup:store-config:set --base-url="https://mg2.ngrok.io/`.
 * Run [`ddev share`](../usage/commands.md#share) and you’ll see your project at `mg2.ngrok.io`.
 
+## Using localhost.run service
+
+* Another solution is to use [`https://localhost.run/`](https://localhost.run/) . It provides a proxy service to point a domain name to your localhost without setting up or installing any packages.
+* All you need to do is identify the **http** port of the ddev server, which can be found by executing `ddev describe` and run
+`ssh -R 80:localhost:PORT localhost.run` you will get a publicly accessible URL.
+* Make sure you have one ssh key in your local (no need to add anywhere separately it will prompt to fingerprint on first run).
+* If you wish to use [`Custom Domains`](https://localhost.run/docs/custom-domains) they have a paid plan.
```

Comment:
I’d start with at least an introductory sentence before jumping into bullet points. (Ideally those would highlight tips or sequential steps.)

Example with stylistic cleanup for consistency and distinct steps someone could follow:

```suggestion
[https://localhost.run/](https://localhost.run/) is a proxy service that points a domain name to your localhost without setting up or installing any packages. The only thing you’ll need to provide is the **HTTP** port of the DDEV web server.

1. Run `ddev describe` and look for the project’s `http://` URL in the “All URLs” row.
2. Run `ssh -R 80:localhost:PORT localhost.run` to get a publicly-accessible URL.
3. Make sure you have one SSH key in your workstation’s keyring. (No need to specify it anywhere; localhost.run will prompt to fingerprint when you first run it.)

You can optionally use [custom domains](https://localhost.run/docs/custom-domains) with a paid plan.
```

I probably got something wrong, because I’m not sure where you intend the URL from step 1 to be used. Correct as necessary!

Reply:
I found that https://localhost.run/docs/ has better description about the service so I reused the same here. Please make edits if any found any improvement needed.

Reply:
> I’m not sure where you intend the URL from step 1 to be used.
Can you elaborate it ?

Reply:
> I found that https://localhost.run/docs/ has better description about the service so I reused the same here. Please make edits if you think any improvement needed.

I suggested edits in [that comment](https://github.com/ddev/ddev/pull/5173#discussion_r1269883925) and described why I would make them. You’re welcome to use or ignore whatever you’d like!

> Can you elaborate it ?
> 
> We are not using URL anywhere, but to identify HTTP port.

Got it. I was struggling to make the connection between whatever I was supposed to get from `ddev describe` and where that fits into `ssh -R 80:localhost:PORT localhost.run`. If I’m understanding you correctly, when I run `ddev describe` I need to be looking for the HTTP “Host:” port in the “web” row:

![Screen Shot 2023-07-21 at 08 18 03 AM@2x](https://github.com/ddev/ddev/assets/2488775/2c7b6e1e-f6f9-41aa-8b1e-dac700f5acd8)

In this case, I needed the second one (`63837`)—and then that needed to be substituted for `PORT` in your example. I updated my suggestion above to try and clarify the relationship since that’s the crucial part.

Reply:
@mattstein You don't need to identify it , its already available in `ddev describe` command under **All URLs**. Refer below image.
<img width="677" alt="ddev-describe" src="https://github.com/ddev/ddev/assets/8525040/5dfbba23-8bcf-4db3-b269-1b53a6fde1d3">

Reply:
@bhanu951 Great! The image is helpful; anything to make what to look for obvious. Thank you!